pitch.md

```diff
@@ -46,24 +92,34 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There aren't any other DSLs in this domain that I know of. Automated coffee machines already have a GUI that lets you customize your drink in this way but
+
+1. It almost certainly doesn't compile to a DSL (I guess **maybe** it does, but I highly doubt it).
+2. The "language" has limited expressiveness. You can choose a base drink, and then modify the amount of coffee, sugar, and milk to add, but that's about it. The language I'm proposing would allow you to create a drink from the bottom up. 
```

Comment:
As we discussed, it might be helpful to look into different types of coffee machines available and the amount of customization they currently offer.